homu/server.py

```diff
@@ -242,7 +255,7 @@ def github():
     payload = request.body.read()
     info = request.json
 
-    lazy_debug(logger, lambda: 'info: {}'.format(utils.remove_url_keys_from_json(info)))
+    lazy_debug(logger, lambda: 'info: {}'.format(utils.remove_url_keys_from_json(info)))  # NOQA
```

Comment:
Good catch on the `# noqa`; I'll change those and use the more-legible variant in the future.

I'm doing basically the same thing to the rest of Homu this week, so that https://github.com/servo/homu/blob/master/.travis.yml#L10 in the Travis config can lose its `--ignore E501`.